tests/storage/business/test_arealink_manager.py

```diff
@@ -134,8 +136,16 @@ def test_area_crud(empty_study: FileStudy, matrix_service: SimpleMatrixService):
     }
 
     area_manager.create_area(study, AreaCreationDTO(name="test2", type=AreaType.AREA))
-    link_manager.create_link(study, LinkInfoDTO(area1="test", area2="test2"))
+    study.version = 820
+    link_manager.create_link(
+        study,
+        LinkDTO(
+            area1="test",
+            area2="test2",
+        ),
+    )
     assert empty_study.config.areas["test"].links.get("test2") is not None
+    study.version = -1
```

Comment:
Why don't use simply put the version at 820 at the beginning and don't change it ?

Reply:
Yes